UnityProject/Assets/Scripts/Messages/Client/VariableViewer/RequestChangeVariableNetMessage.cs

```diff
@@ -20,42 +21,22 @@ void ValidateAdmin()
 	{
 		var admin = PlayerList.Instance.GetAdmin(AdminId, AdminToken);
 		if (admin == null) return;
-		VariableViewer.RequestChangeVariable(PageID, newValue);
+		VariableViewer.RequestChangeVariable(PageID, newValue,SendToClient, SentByPlayer.GameObject, AdminId);
 
 		Logger.Log($"Admin {admin.name} changed variable {PageID} (in VV) with a new value of: {newValue} ",
 			Category.Admin);
 	}
 
 
-	public static RequestChangeVariableNetMessage Send(ulong _PageID, string _newValue, string adminId, string adminToken)
+	public static RequestChangeVariableNetMessage Send(ulong _PageID, string _newValue ,bool InSendToClient , string adminId, string adminToken)
```

Comment:
Why do you return the message? Rider says the return value is never used so this could be changed
```suggestion
	public static void Send(ulong _PageID, string _newValue ,bool InSendToClient , string adminId, string adminToken)
```